### ISSUE TITLE
STAR

### ISSUE THREAD

**Issue by mdlong-rpccc** [ubccr/software-layer#323]
Can you please install the latest version of STAR as a module on CCR - the latest version I see on easybuild is 2.7.9a (https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/s/STAR)

Thanks,
Mark

**Comment by tonykew** [comment]
I've completed a test build of STAR version 2.7.11b which is the most recent
release version from here:  https://github.com/alexdobin/STAR/releases
...but I have no test code to verify that it is working properly.

**Comment by tonykew** [comment]
STAR version 2.7.11b has been published to the ccrsoft/2023.01 software release


```
login1$ module spider star

----------------------------------------------------------------------------
  star:
----------------------------------------------------------------------------
    Description:
      STAR aligns RNA-seq reads to a reference genome using uncompressed
      suffix arrays.

     Versions:
        star/2.7.9a
        star/2.7.11b
[...]
login1$ module spider star/2.7.11b

----------------------------------------------------------------------------
  star: star/2.7.11b
----------------------------------------------------------------------------
    Description:
      STAR aligns RNA-seq reads to a reference genome using uncompressed
      suffix arrays.

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "star/2.7.11b" module is available to load.

      gcc/11.2.0
[...]
login1$ module load gcc/11.2.0 star/2.7.11b
login1$ 
```

This module includes two binaries, STAR and STARlong:

```
login1$ ls ${EBROOTSTAR}/bin
STAR  STARlong
login1$ 
login1$ STAR
Usage: STAR  [options]... --genomeDir /path/to/genome/index/   --readFilesIn R1.fq R2.fq
Spliced Transcripts Alignment to a Reference (c) Alexander Dobin, 2009-2022

STAR version=2.7.11b
STAR compilation time,server,dir=2024-06-06T15:35:40-04:00 :/scratch/tonykew_STAR-2.7.11b-GCC-11.2.0.eb/tmp/tonykew/easybuild/build/STAR/2.7.11b/GCC-11.2.0/STAR-2.7.11b/source
For more details see:
<https://github.com/alexdobin/STAR>
<https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf>

To list all parameters, run STAR --help
login1$ 
login1$ STARlong 
Usage: STAR  [options]... --genomeDir /path/to/genome/index/   --readFilesIn R1.fq R2.fq
Spliced Transcripts Alignment to a Reference (c) Alexander Dobin, 2009-2022

STAR version=2.7.11b
STAR compilation time,server,dir=2024-06-06T15:35:40-04:00 :/scratch/tonykew_STAR-2.7.11b-GCC-11.2.0.eb/tmp/tonykew/easybuild/build/STAR/2.7.11b/GCC-11.2.0/STAR-2.7.11b/source
For more details see:
<https://github.com/alexdobin/STAR>
<https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf>

To list all parameters, run STAR --help
login1$ 

```
